.github/workflows/stale.yaml

```diff
@@ -0,0 +1,32 @@
+name: 'Stale issue handler'
+on:
+  workflow_dispatch:
+  schedule:
+    # The start of everyday
+    - cron: '0 0 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v5.1.1
+        id: stale
+        with:
+          # PR configuration
+          days-before-pr-stale: 30
+          stale-pr-message: 'This PR has become stale because it has been open for 30 days with no activity. Please provide an update, merge, close, or add the lifecycle/frozen label to this PR before it is automatically closed.
```

Comment:
```suggestion
          stale-pr-message: 'This PR has become stale because it has been open for 30 days with no activity. Please provide an update, merge, close, or add the lifecycle/frozen label to this PR before it is automatically closed.'
```